js/flightlog_fields_presenter.js

```diff
@@ -1056,7 +1081,15 @@ function FlightLogFieldPresenter() {
                             return value.toFixed(0) + "Hz";
                     }
                 case 'GPS_RESCUE_THROTTLE_PID':
-                        return value.toFixed(0);
+                    switch (fieldName) {
+                        // debug 0 = Throttle P
+                        // debug 1 = Throttle I
+                        // debug 3 = Throttle D
+                        case 'debug[3]': // current altitude in m
+                            return (value / 100).toFixed(2) + 'm';
+                        default:
+                            return value.toFixed(0);
+                    }
```

Comment:
I suppose that `debug 3` is wrong and is `debug 2`. Move the comments to the default to let clear where it will enter. For example:
```suggestion
                    switch (fieldName) {
                        case 'debug[3]': // current altitude in m
                            return (value / 100).toFixed(2) + 'm';                            

                        // debug 0 = Throttle P
                        // debug 1 = Throttle I
                        // debug 2 = Throttle D                            
                        default:
                            return value.toFixed(0);
                    }
```

Reply:
Thanks, have done as suggested.